### PR TITLE
 Ballot Use Cases (Ballot# 1, Ballot# 132, Ballot# 102) #6, #134, #107. #39

### PR DESCRIPTION
**Pull request by Healthedata1**

### CODE REVIEW

File: spec/index.md [hunk 1]
```diff
@@ -4,11 +4,7 @@ layout: default
 ---
 {% include publish-box.md %}
 
-SMART on FHIR provides reliable, secure authorization for a variety of app
-architectures through the use of the OAuth 2.0 standard.  The Launch Framework
-supports the [four uses cases](http://argonautwiki.hl7.org/images/4/4c/Argonaut_UseCasesV1.pdf)
-defined for Phase 1 of the [Argonaut
-Project](http://argonautwiki.hl7.org/index.php?title=Main_Page).  
+The SMART App Launch Framework connects third-party applications to Electronic Health Record data, allowing apps to launch from inside or outside the user interface of an EHR system. The framework supports apps for use by clinicians, patients, and others. It provides a reliable, secure authorization protocol for a variety of app architectures, including apps that run on an end-user's device as well as apps that run on a secure server.  The Launch Framework supports the [four uses cases](http://argonautwiki.hl7.org/images/4/4c/Argonaut_UseCasesV1.pdf) defined for Phase 1 of the [ArgonautProject](http://argonautwiki.hl7.org/index.php?title=Main_Page).
```
Comment (jmandel) [inline spec/index.md]: summarize the four use cases as: "Patients apps that launch standalone; Patient apps that launch from a portal; Provider apps that launch standalone; Provider apps that launch from a portal".

Comment (jmandel) [inline spec/index.md]: Maybe summarize the four use cases as: "Patients apps that launch standalone; Patient apps that launch from a portal; Provider apps that launch standalone; Provider apps that launch from a portal".

**Comment by Healthedata1** [comment]
testing to see if notification sent

**Comment by jmandel** [comment]
Addressed by https://github.com/HL7/smart-app-launch/commit/3c0bdbffb74a37713723e643ff314c20006b5001

**Pull request closed without merging**